app/serializers/api/v2/serializable_course.rb

```diff
@@ -67,7 +67,7 @@ def enrichment_attribute(name, enrichment_name = name)
 
       meta do
         {
-          edit_options: @object.edit_options.all
+          edit_options: edit_course_options(@object)
```

Comment:
Should be able to do `object.edit_course_options` instead.

Reply:
Done :+1 thanks.

Reply:
Done. Thanks 👍